app/src/lib/progress/git.ts

```diff
@@ -185,13 +199,6 @@ export class GitProgressParser {
     }))
   }
 
-  /**
-   * Parse the given line of output from Git, returns either an IGitProgress
-   * instance if the line could successfully be parsed as a Git progress
-   * event whose title was registered with this parser or an IGitOutput
-   * instance if the line couldn't be parsed or if the title wasn't
-   * registered with the parser.
-   */
```

Comment:
I think we should keep this documentation here and modify the documentation for the interface version of this. A GitProgressParser was specifically intended to parse the well-defined progress format of Git commands which supports `--progress` whereas the rebase progress parser parses a custom format. As such the `steps` concepts in this concrete class doesn't apply to the rebase parser and consequently shouldn't be described in the interface either. So I suggest reinstating this and I'll add a suggested change for the interface documentation.

Reply:
This has been addressed in bfe7846e7